scripts/!mods_preload/mod_reforged.nut

```diff
@@ -39,6 +39,32 @@ foreach (requirement in requiredMods)
 	queueLoadOrder.push(">" + (idx == null ? requirement : requirement.slice(0, idx)));
 }
 
+// TODO: Establish a cleaner way to organize "Early" bucket hooks on a per-file basis
+::Reforged.HooksMod.queue(queueLoadOrder, function() {
+	::Reforged.HooksMod.hookTree("scripts/entity/tactical/actor", function(q) {
+		q.onDeath = @(__original) function( _killer, _skill, _tile, _fatalityType )
+		{
+			if (_killer != null && _killer.getFaction() != ::Const.Faction.Player && _killer.getFaction() != ::Const.Faction.PlayerAnimals)
```

Comment:
Does this line mean that a last-hit still always guarantees the loot for the player?
As I understand it, you only turn non-player kills into null-kills with that new loot-mechanic so that the player overall gets loot in more situations.
But you don't change player-/null-kills into ally/enemy-kills in order to prevent the player from kill-stealing, like during early game cheese where you fight alongside a mercenary/noble army.

Is that observation correct?
If so, was that intended, are there any plans to fix that?

Reply:
After your comment, we talked about it in Discord and decided that we don't want this cheese and only want to drop the loot if the player did at least 50% of the total damage. So I have added a commit which reverts this change.